### DISCUSSION TITLE
Looking for help?

### DISCUSSION THREAD

**Discussion by Spleeding1** [wprig/wprig#751] [General]
@robruiz I was wondering if I would be able to help contribute to this project. I have come to be dependent on it for theme building and am concerned about the out of date errors that show up.

**Comment by heathernew09** [comment]
We would love your help, to start would you like an invite to the slack channel @Spleeding1 ?

**Reply by Spleeding1** [reply]
Yeah, sure. Let me know if you need any info @heathernew09

**Reply by heathernew09** [reply]
@Spleeding1 what's your email? I'll send an invite

**Reply by Spleeding1** [reply]
@heathernew09 developing.rust@gmail.com
